### DISCUSSION TITLE
make fish integration work with zoxide cd alias

### DISCUSSION THREAD

**Discussion by JohnFinn** [sxyazi/yazi#1595] [4. Ideas]
Fish integration here https://yazi-rs.github.io/docs/quick-start/ can be modified by adding `builtin` command before `cd` to make it work with `zoxide init fish --cmd cd | source`
```fish
function yy
	set tmp (mktemp -t "yazi-cwd.XXXXXX")
	yazi $argv --cwd-file="$tmp"
	if set cwd (command cat -- "$tmp"); and [ -n "$cwd" ]; and [ "$cwd" != "$PWD" ]
		builtin cd -- "$cwd"
	end
	rm -f -- "$tmp"
end
```
I couldn't find the code location to make a PR. So opening a discussion.
There already exists the discussion, but it's locked https://github.com/sxyazi/yazi/discussions/1287

**Comment by AnirudhG07** [comment]
This will be added to the [yazi-rs](https://github.com/yazi-rs/yazi-rs.github.io) github repository. This repository contains only the source code for yazi, the documentations are all there.
In your case, you would wanna add it [here](https://github.com/yazi-rs/yazi-rs.github.io/blob/main/docs/quick-start.md).

**Reply by JohnFinn** [reply]
Thanks! Made a PR https://github.com/yazi-rs/yazi-rs.github.io/pull/135
